### ISSUE TITLE
Initial subscribe email is sent form verificationemail.com

### ISSUE THREAD

**Issue by jloh** [ks888/LambStatus#122]
Not sure if this is intended or not, but after setting up email notifications with a custom source email address the original subscribe email is still sent from `no-reply@verificationemail.com`.

The emails after that for the actual notifications are indeed sent from the correct email address however.

**Comment by ks888** [comment]
Yes, it's definitely better to use custom email address there, but I'm not very sure how to do that. [This doc](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pool-settings-email-address-customization.html) says *the SES-verified email address* can be specified as FROM email address, but no descriptions about *the SES-verified domain*. As the aws console has the UI only for the SES-verified email address (like below), maybe there is no way to do that so far?

<img width="839" alt="screen shot 2018-06-02 at 23 38 00" src="https://user-images.githubusercontent.com/8448120/40877568-08836db6-66be-11e8-8f9d-67cced67abdb.png">
